app/views/layouts/nav/_project.html.haml

```diff
@@ -44,11 +44,12 @@
           %span
             Graphs
 
-    = nav_link(controller: :milestones) do
-      = link_to namespace_project_milestones_path(@project.namespace, @project), title: 'Milestones' do
-        %i.fa.fa-clock-o
-        %span
-          Milestones
+    - if project_nav_tab? :issues
```

Comment:
Please use `project_nav_tab? :milestones` here, and modify the `get_project_nav_tabs` helper method to include `:milesones` `if project.issues_enabled`.